python-3.12.yaml

```diff
@@ -52,7 +52,8 @@ pipeline:
       repository: https://github.com/python/cpython.git
       tag: v${{package.version}}
       cherry-picks: |
-        3.12/dcc5182f27c1500006a1ef78e10613bb45788dea:  CVE-2024-8088 fixes
+        3.12/dcc5182f27c1500006a1ef78e10613bb45788dea: CVE-2024-8088 fixes
+        3.12/dcc3eaef98cd94d6cb6cb0f44bd1c903d04f33b1: CVE-2024-7592 fixes
```

Comment:
random thing. wow. am I right that the odds of 2 commits listed here both colliding on the first 3 chars are 1/4096 ? it was enough to make me double take at least.